lib/net.js

```diff
@@ -578,7 +578,7 @@ Socket.prototype._destroy = function(exception, cb) {
   if (this._handle) {
     if (this !== process.stderr)
       debug('close handle');
-    var isException = exception ? true : false;
+    var isException = !!exception;
```

Comment:
This could become `const` while you're here.

Reply:
ok